docs/docs.json

```diff
@@ -482,5 +482,282 @@
+			"live": "✘",
+			"path": "/docs/ui/#bar",
+			"link": "UI-Customization"
+		},
+		{
+			"attribute": "mv-path",
+			"on": "Properties",
+			"purpose": "Render and edit only a subset of the stored data.",
+			"live": "Initially"
+		},
+		{
+			"attribute": "mv-alias",
+			"on": "Properties",
+			"purpose": "If provided, Mavo will read data from this property **instead** of its real property name. Useful to read stored data properly after renaming a property in the HTML.",
+			"live": "✘",
+			"path": "/docs/migration/#mv-alias",
+			"link": "Migration"
+		}
+	],
+	"class": [
+		{
+			"name": "mv-autoedit",
+			"on": "Mavo root",
+			"purpose": "Mavo root\tMavo will start in edit mode (no need to click the Edit button). Useful for apps that are almost always in edit mode, but still need a read mode (if read mode is not needed at all, use `mv-mode` instead)."
+		},
+		{
+			"name": "mv-fixed",
+			"on": "Mavo toolbar",
+			"purpose": "Style the toolbar as fixed at the top of the screen (always visible)."
+		},
+		{
+			"name": "mv-logged-in",
+			"purpose": "Only show element to logged in users."
+		},
+		{
+			"name": "mv-logged-out",
+			"purpose": "Only show element to logged out users."
+		}
+	],
+	"generated-attribute": [
+		{
+			"name": "mv-progress",
+			"on": "Mavo root",
+			"purpose": "Only set when an event is in progress. Its value reflects what event, e.g. `\"Saving\"` or `\"Loading\"`."
+		},
+		{
+			"name": "mv-clicked",
+			"on": "Properties that are `<button>` elements or have a `counter` class.",
+			"purpose": "The number of times the button has been clicked."
+		},
+		{
+			"name": "mv-permissions",
+			"on": "Mavo root",
+			"purpose": "Space-separated list of permissions (read, edit, add, delete, login, logout, save)"
+		},
+		{
+			"name": "mv-mode",
+			"on": "Any property",
+			"purpose": "`mv-mode=\"edit\"` is applied on every property when it’s being edited."
+		}
+	],
+	"generated-class": [
+		{
+			"name": "mv-target-within",
+			"set-on": "Any element that matches the CSS `:target` pseudo-class or contains an element that does."
+		},
+		{
+			"name": "mv-ui",
+			"set-on": "Any (usually generated) controls for triggering Mavo-related actions"
+		},
+		{
+			"name": "mv-empty",
+			"set-on": "Properties with no value"
+		},
+		{
+			"name": "mv-deleted",
+			"set-on": "Collection items that have been deleted but the deletion is not yet saved"
+		},
+		{
+			"name": "mv-item-bar",
+			"set-on": "Buttons on collection items for deleting, adding, rearranging."
+		},
+		{
+			"name": "mv-highlight",
+			"set-on": "Highlighted collection items (e.g. when hovering over the delete button)"
+		},
+		{
+			"name": "mv-bar",
+			"set-on": "Mavo toolbar"
+		},
+		{
+			"name": "mv-editor",
+			"set-on": "Element used for editing a property"
+		},
+		{
+			"name": "mv-popup",
+			"set-on": "Generated popup for editing an attribute."
+		}
+	],
+	"css-attribute": [
```

Comment:
These are CSS variables or CSS custom properties. There is no such thing as a CSS attribute.